recipes/1-Insights/conditioned-insights.ipynb

```diff
@@ -0,0 +1,309 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Conditioned Insights\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "Howso Engine enables powerful prediction insights through Howso's multitude of prediction stats views. includes a variety of ways to view and condition condition statistcs\n",
```

Comment:
What is a prediction stat view? Can we rephrase this?

Reply:
Also, the sentence is a bit weird... I think it should be: " Howso Engine enables powerful prediction insights through many prediction stats views [with an explanation of what this is], including tools to view and condition the statistics.

Reply:
Then, we need an explanation of why this matters. For example, "An understanding of how your dataset's predictability changes under a variety of conditions (feature addition/removal, case addition/removal) provides out of the box insight in to learning decisions without the need to build multiple ML models to test your theories.".... or something along those lines